packages/cmcd-validator-library/src/inputValidator/configValidator/checkConfig.js

```diff
@@ -0,0 +1,34 @@
+import { cmcdTypes, keyTypes, errorTypes, cmcdHeader } from '../../utils/constants.js';
+import { createError } from '../../utils/error.js';
+
+export const checkConfig = (config, errors) => {
+  const { customKey, specificKey } = config;
+  if (customKey) {
+    const types = Object.values(cmcdTypes);
+    const headers = Object.keys(cmcdHeader);
+    customKey.forEach((customObj) => {
+      if (!(/^.{0,}[^-]-[^-].{0,}$/g.test(customObj.key))) {
```

Comment:
This regular expresion it is no totally correct. For example, --hello-world-- is correct using this regex

Reply:
thanks, it was fix